docs/getting-started/core-concepts.md

```diff
@@ -25,3 +25,4 @@ In some other cases, you might want to instantiate Wallet from another input suc
 - `privateKey`: This allow to manage a single privateKey/publicKey set. Therefore, you will only have a single unique address to receive money. 
 - `HDPublicKey`: This allow a "watch-only" mode. You won't be able to spend anything, but this will allow you to track and monitor in real-time the addresses set of this public key. Use-case might be to always pay to another new 
 
+[Learn more about instantiation documentation](/wallet/Wallet.md)
```

Comment:
Is `/wallet/Wallet.md` correct? It doesn't work if you click it when viewing the md, but maybe it does when docsify builds the docs?

Reply:
Yes, this is an issue we had with a PR suggested by pasta in the past. 
We can make a direct link to the full path, but if we go this route, it requires to be done for all the similar link. 

What do you think @thephez ? I do find some good reason to provide it for when we actually browse from github (not on github.io), in the other end, if one day we get rid of github.io, it will not work, and it makes it less clear for when we browse on an offline condition 🤔

Reply:
I think it's fine to leave it the way that works on the github.io page. That's the preferred place to look at it. 

Also, it's easy enough to view docsify docs locally. Just run:
`cd docs && python -m SimpleHTTPServer 3000` 
from the repo root as described in the [docsify quickstart](https://docsify.js.org/#/quickstart?id=manual-initialization). Another offline option would be to look at the [PWA config](https://docsify.js.org/#/pwa).